### PR TITLE
Remove UserAgent output and redirect to WPT in the README.

### PR DESCRIPTION
**Pull request by fred-wang**
No need to maintain this duplicate copy.

cc @khaledhosny

### CODE REVIEW

**Comment by khaledhosny** [comment]
That is up to @rsheeter or @svgeesus to decide.

**Comment by fred-wang** [comment]
To explain the rationale:

* The WPT version is important for automated testing in browser vendor and W3C repositories and should be regularly kept updated against the woff2-tests generators. woff2-compiled-tests was last updated 6 months ago while after all my PRs are reviewed and merged, the woff2-tests and wpt will be in sync.

* Duplication adds confusion for people willing to use or edit the lastest version WPT tests. We already have 3 different repos. If we reduce them to only two + add the DO NOT EDIT warning, that would already be much better. See https://github.com/web-platform-tests/wpt/issues/9300

cc @foolip

**Review by svgeesus — APPROVED** [review]
Agree, now the compiled output is in WPT we no longer need this one.

**Pull request merged**
